### PR TITLE
Add key bindings for up and down arrows, home and end

### PR DESCRIPTION
**Pull request by AngelMerinoB**
Hello @timwright12,

this PR pretends to address #5 

I implemented it based on the w3 specifications that you shared, but I'm open to any suggestions. 

I also replaced `accordionHeading.focus()` in line 115 from src/accordion.js to focus the header item instead of the heading from the content panel as it seems to happen in the w3 example. 

Since I'm working on Windows I had to disable eslint for the linebreak-style that differ from MacOs or Linux. I hope it causes no problem.

I'm still learning, so I'm open to any feedback on this.

### CODE REVIEW

**Comment by timwright12** [comment]
Thanks @AngelMerinoB! cc'ing @garand on this.

**Review by kdo — CHANGES_REQUESTED** [review]
@AngelMerinoB Thanks for tackling this! Looks good to me, except the `/* eslint-disable linebreak-style */` comments, can you remove them? If you get errors, I think `linebreak-style` can be fixed with the cli. Thanks!

File: src/accordion.js [hunk 1]
```diff
@@ -69,6 +70,15 @@ export default class Accordion {
 		let accordionLinks = accordionArea.querySelectorAll( '.accordion-header' );
 		let accordionContent = accordionArea.querySelectorAll( '.accordion-content' );
 
+		// Handle keydown event to move between accordion items
+		accordionArea.addEventListener( 'keydown', () => {
```
Comment (kdo) [inline src/accordion.js]: @AngelMerinoB missing the event parameter, causing a bug on FF

**Comment by AngelMerinoB** [comment]
> Thanks @AngelMerinoB! cc'ing @garand on this.

File: src/accordion.js (hunk 1 above)
Comment (AngelMerinoB) [reply src/accordion.js]: Hello @kdo ,
I already commited the changes, it works now on FF. The focus outline is not appearing however (maybe that is more related to CSS) but you can test it using the enter key. I also deleted the extra "eslint-disable" lines. 

Sorry about the closed and reopened PR action a few minutes ago. If there is anything else to modify I'd be glad to help.

**Review by kdo — APPROVED** [review]

**Comment by timwright12** [comment]
@kdo this also needs a version bump to be published to npm

**Pull request merged**